### ISSUE TITLE
Implement all parameters from `SGDRegressor`

### ISSUE THREAD

**Issue by TechSolomon** [jbonda/ml-workflow-pipeline#34]
Source: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html

- [x] loss
- [x] penalty
- [x] alpha 
- [x] l1_ratio 
- [x] fit_intercept 
- [x] max_iter 
- [x] tol 
- [x] shuffle 
- [x] verbose 
- [x] epsilon 
- [x] random_state 
- [x] learning_rate
- [x] eta0 
- [x] power_t 
- [x] early_stopping 
- [x] validation_fraction 
- [x] n_iter_no_change 
- [x] warm_start 
- [x] average
